src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -616,6 +624,19 @@ int getXpUntilNextTile() {
         return xpUntilNextTile;
     }
 
+    private boolean modeTilescontainSelectedSceneTile(){
```

Comment:
A few things, if you could rename to 'isTileMarked' and add a parameter 'tile'. Just means we can use this elsewhere if we want to know if any tile is marked.